assets/config/Dashboard.Global.json

```diff
@@ -0,0 +1,69 @@
+{
```

Comment:
This file is not supposed to be in `assets` folder, `assets` folder will be packed into the binary, but we should allow the user to provide their own `json` template, so I suggest put this file into an `example` folder in the root directory or docs directory, and give an example command in the doc

Reply:
How about we provide a default `json` template, which is packed into the binary. Only when users want to customize, they need to provide their own template. Otherwise, this command is not so easy to use.

Reply:
Sounds good to me